docs/build/build-transaction-construction.md

```diff
@@ -35,6 +35,29 @@ function from the Balances pallet will take:
 - `dest`: Destination address
 - `#[compact] value`: Number of tokens (compact encoding)
 
+**Serialized transaction format**
+
+Before being submitted, transactions are serialized. Serialized transactions are hex encoded SCALE-encoded bytes. The specific serialization is defined in the runtime and can change if the runtime is upgraded, but in general the serialization format can be described as follows:
+- Compact encoded number of SCALE-encoded bytes following this.
+- 1 bit: it is a 0 if no signature is present, or a 1 if it is.
+- 7 bits: the extrinsic version, it is equal to 4 in decimal.
+- If there is a signature:
+  - a SCALE encoded `sp_runtime::MultiAddress::Id<AccountId32, u32>` indicating the signer(s) of the transaction.
+  - a SCALE encoded `sp_runtime::MultiSignature::{SigningScheme}` with the signature\*.
+  - a SCALE encoded `sp_runtime::generic::Era` indicating for how long this transaction is valid.
```

Comment:
Can @Imod7 or @tarikgul comment on encoding for mortal vs. immortal transactions?

Reply:
For encoding between an `Immortal` and `Mortal` era, the following `impl` is useful: https://github.com/paritytech/substrate/blob/master/primitives/runtime/src/generic/era.rs#L102

An immortal era will default to the encoded value of `0x00`. It will be just a single byte of `0`.

A mortal era on the other hand will be `Vec[u64, u64]`; Comprising of the Period, and Phase.